diff --git a/node_modules/alova/dist/alova.esm.js b/node_modules/alova/dist/alova.esm.js
index 42d8f29..68cb594 100644
--- a/node_modules/alova/dist/alova.esm.js
+++ b/node_modules/alova/dist/alova.esm.js
@@ -1362,13 +1362,6 @@ const createAlova = (options) => {
     pushItem(alovas, alovaInstance);
     return alovaInstance;
 };
-/* c8 ignore start */
-if (process.env.NODE_ENV === 'development') {
-    console.log(`powerful alova request strategies: https://alova.js.org/category/strategy
-use mock data: https://alova.js.org/extension/alova-mock
-please star alova if you like it: https://github.com/alovajs/alova
-this tips will be removed in production environment`);
-}
-/* c8 ignore stop */
 
 export { Method, createAlova, key as getMethodKey, invalidateCache, matchSnapshotMethod, queryCache, setCache, updateState, useFetcher, useRequest, useWatcher };
+
